modules/core/src/main/java/org/apache/ignite/internal/managers/systemview/GridSystemViewManager.java

```diff
@@ -75,7 +82,26 @@
      * @param ctx Kernal context.
      */
     public GridSystemViewManager(GridKernalContext ctx) {
-        super(ctx, ctx.config().getSystemViewExporterSpi());
+        super(ctx, ((Supplier<SystemViewExporterSpi[]>)() -> {
```

Comment:
Could you please replace supplier by simple method call? There is no need to use lambda syntax here.

Reply:
replaced.